packages/prepo-shared-contracts/test/SafeAccessControlEnumerable.test.ts

```diff
@@ -0,0 +1,211 @@
+import { expect } from 'chai'
+import { ethers } from 'hardhat'
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/dist/src/signer-with-address'
+import { DEFAULT_ADMIN_ROLE } from 'prepo-constants'
+import { safeAccessControlEnumerableFixture } from './fixtures/SafeAccessControlEnumerableFixture'
+import { SafeAccessControlEnumerable } from '../types/generated'
+import { formatBytes32String } from 'ethers/lib/utils'
+
+describe('SafeAccessControlEnumerable', () => {
+  let deployer: SignerWithAddress
+  let owner: SignerWithAddress
+  let user1: SignerWithAddress
+  let user2: SignerWithAddress
+  let roleAMember: SignerWithAddress
+  let safeAccessControlEnumerable: SafeAccessControlEnumerable
+  const roleA = formatBytes32String('A')
+  const roleB = formatBytes32String('B')
+  const ZERO_HASH = formatBytes32String('')
+
+  const deploySafeAccessControlEnumerable = async (): Promise<void> => {
+    ;[deployer, user1, user2] = await ethers.getSigners()
+    owner = deployer
+    safeAccessControlEnumerable = await safeAccessControlEnumerableFixture()
+  }
+
+  const setupSafeAccessControlEnumerable = async (): Promise<void> => {
+    await deploySafeAccessControlEnumerable()
+    await safeAccessControlEnumerable.connect(deployer).grantRole(DEFAULT_ADMIN_ROLE, owner.address)
+    await safeAccessControlEnumerable.connect(owner).acceptRole(DEFAULT_ADMIN_ROLE)
+  }
+
+  describe('initial state', () => {
+    beforeEach(async () => {
+      await deploySafeAccessControlEnumerable()
+    })
+
+    it('sets default admin to deployer', async () => {
+      expect(await safeAccessControlEnumerable.hasRole(DEFAULT_ADMIN_ROLE, deployer.address)).to.eq(
+        true
+      )
+    })
+  })
+
+  describe('# setRoleAdminNominee', () => {
+    beforeEach(async () => {
+      roleAMember = user1
+      await setupSafeAccessControlEnumerable()
+      await safeAccessControlEnumerable.connect(owner).grantRole(roleA, roleAMember.address)
+      await safeAccessControlEnumerable.connect(roleAMember).acceptRole(roleA)
+      expect(await safeAccessControlEnumerable.hasRole(roleA, roleAMember.address)).to.eq(true)
+    })
+
+    it('reverts if not role admin and not role admin nominee', async () => {
```

Comment:
```suggestion
    it('reverts if not role admin nominee and not role admin', async () => {
```
Let's stick with consistent ordering